token/js/test/unit/transferfee.test.ts

```diff
@@ -1,98 +1,167 @@
-import { TransferFee, TransferFeeConfig, calculateFee, calculateEpochFee, ONE_IN_BASIS_POINTS } from '../../src';
+import {
+    calculateFee,
+    calculateEpochFee,
+    ONE_IN_BASIS_POINTS,
+    createInitializeTransferFeeConfigInstruction,
+    decodeInitializeTransferFeeConfigInstructionUnchecked,
+} from '../../src';
 import { expect } from 'chai';
-import { PublicKey } from '@solana/web3.js';
+import { Keypair, PublicKey } from '@solana/web3.js';
 
-describe('calculateFee', () => {
-    it('should return 0 fee when transferFeeBasisPoints is 0', () => {
-        const transferFee = {
-            epoch: 1n,
-            maximumFee: 100n,
-            transferFeeBasisPoints: 0,
-        };
-        const preFeeAmount = 100n;
-        const fee = calculateFee(transferFee, preFeeAmount);
-        expect(fee).to.eql(0n);
+describe('transferFee', () => {
+    describe('encoding/decoding `InitializeTransferFeeConfig` instructions', () => {
```

Comment:
Test diff here looks awful as well, but here I've just added the `encoding/decoding` section above the existing tests.